rladies/meetupr
# Use `graphql` API

- [x] Use latest GHA #121 
- [x] Bump version #122 
- [x] Fix check warnings #124
- [x] Move GraphQL files to `inst` folder #125
	- `./inst/graphql/NAME.graphql`
- [x] Implement GraphQL pagination (w/ progress bar) #125
- Add support for extra fields 
	- [x] Query the extra fields using `{glue}` #125
	- [x] Dynamically name fields #130 
- [x] Transform GraphQL response data into data.frames #130
- Use new methods
	- [x] `get_events2()` #132 
	- [x] `find_groups2()` #132
	- [x] ~~`get_boards()`~~ Not possible, see #131 
- For `{meetupr}` authors:
	- Make new methods:
		- [x] `get_event_attendees2()`
		- [x] `get_event_comments2()`
		- [x] `get_event_rsvps2()`
		- [x] `get_members2()`
		- [ ] `get_pro2()`
		- [x] `meetup_query2()`. Similar to `graphql_query()`
	- [ ] For all GraphQL implementations, rename `FUNC2()` -> `FUNC()`; Remove original `FUNC()` implementations.
